packages/Algernon.package/ALGWindowType.class/class/algertemList.st

```diff
@@ -0,0 +1,5 @@
+as yet unclassified
```

Comment:
This method and also `visibleWindows` seem to be unclassified at this point. I thought we had a linting test for this but maybe the test for no unclassified methods just covers instance methods? 🤔

Reply:
Hm, very weird. Maybe we should have another look at these linting tests?

Reply:
Ok, I feel like this is a problem with SWALint. And I couldn't find any Lint which checks for class method categorization :/ Maybe you want to have another look at that?

Reply:
Okay, that is unfortunate. Yep, as far as i can see there is no equivalent test for class methods.